### PR TITLE
store/tikv: move 2pc tests to standalone directory

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?
store/tikv has dependencies on `mockstore` and `unistore` because tests need to use them. We can resolve it by moving all tests to standaline package.

Part of https://github.com/pingcap/tidb/issues/22513

### What is changed and how it works?
What's Changed:
- move test files into `store/tikv/tests`
- add *Probe utils to expose internal states of `store/tikv`.

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

File: store/tikv/tests/2pc_slow_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Copyright 2016 PingCAP, Inc.
```
Comment (AndreMouche) [inline store/tikv/tests/2pc_slow_test.go:1]: 2021?

Comment (disksing) [reply store/tikv/tests/2pc_slow_test.go:1]: all code copied from old file.

**Review by AndreMouche — APPROVED** [review]
LGTM

**Review by andylokandy — APPROVED** [review]

**Pull request merged**